### ISSUE TITLE
Hope you can help me

### ISSUE THREAD

**Issue by AjayzzZ4587** [yd-kwon/MatNet#3]
If I met the Multiple data matrices, what should I do， May I do like this ： scores = torch.stack((dot_product_score, cost_mat1, cost_mat2, cost_mat3 ), dim=4).
There may also be papers that I didn't fully understand, forgive me for being a newbie

**Comment by yd-kwon** [comment]
Yes. That is exactly how I have done when I had to deal with multiple data matrices.
Also, when you do so, do not forget to change the shape of "mix1_weight" accordingly.

**Comment by AjayzzZ4587** [comment]
Thanks a million！！！

**Comment by AjayzzZ4587** [comment]
Thanks a million！！！！！




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月23日(星期六) 上午10:47
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [yd-kwon/MatNet] Hope you can help me (Issue #3)





 
Yes. That is exactly how I have done when I had to deal with multiple data matrices.
 Also, when you do so, do not forget to change the shape of "mix1_weight" accordingly.
